.travis.yml

```diff
@@ -18,8 +18,13 @@ before_install:
 
 install:
   - pip install -e .
+  - cd dataset
+  - bash download.sh
+  - cd ..
 
 script:
   - python -c "import keras.backend"
   - sed -i -e 's/"backend":[[:space:]]*"[^"]*/"backend":\ "'$KERAS_BACKEND'/g' ~/.keras/keras.json;
   - echo -e "Running tests with the following config:\n$(cat ~/.keras/keras.json)"
+  - cd pria_lifechem/models
+  - python test_demo.py
```

Comment:
We should leave this as-is for now, but I'm noting that we've discussed using pytest instead, especially if we add more fine-grained unit tests.

Reply:
I will leave this in a separate PR.

Reply:
Sounds good.  And we may decide it isn't necessary to switch to pytest.